showmehow/lessons.json

```diff
@@ -672,15 +672,33 @@
         "entry": "add_numbers",
         "practice": {
             "add_numbers": {
-                "task": "Arithmetic is understood in most programming languages like Python. Try assigning 'a' using the '=' operator to 4 + 7.",
+                "task": "Arithmetic is understood in most programming languages like Python. Try typing '4 + 7'.",
```

Comment:
I would suggest: `Arithmetic is understood in most programming languages, including Python.`